scripts/merge_loh.R

```diff
@@ -12,7 +12,27 @@ Options:
 library(docopt)
 library(dplyr)
 library(GenomicRanges)
-source("scripts/merge_mafs.R")
+
+# Use script-relative import replacement for
+#   source("scripts/merge_mafs.R")
+
+# Find local script - method 1
+library(tidyverse)
+get_this_file <- function(){
+    commandArgs() %>%
+       tibble::enframe(name=NULL) %>%
+       tidyr::separate(col=value, into=c("key", "value"), sep="=", fill='right') %>%
+       dplyr::filter(key == "--file") %>%
+       dplyr::pull(value)
+}
+this_file <- get_this_file()
+source(paste(dirname(this_file), "merge_mafs.R", sep="/"))
+
+# Find local script - method 2
+# Require install of new library funr
+# this_folder <- funr::get_script_path()
+# source(paste(this_folder, "merge_mafs.R", sep="/"))
+
```

Comment:
This section is the most questionable parts of the changes.

Reply:
This seemed like the nicest method of doing this import, but a new  R package was needed.

Reply:
Instead of sourcing merge_mafs.R, I'm pretty sure we could do away with it and just add the functions to merge_loh.R?

Reply:
I like the idea of just adding the function to merge_loh.R
I'll try it.
I take it merge_mafs.R does not need to be a separate script as far as you can remember?